src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Consider removing the eslint-disable rule for '@typescript-eslint/no-explicit-any' and use a more specific type instead of 'any' for better type safety. 😊